packages/open-api-gateway/src/project/open-api-gateway-ts-project.ts

```diff
@@ -57,22 +73,42 @@ export class OpenApiGatewayTsProject extends TypeScriptProject {
    */
   public readonly generatedTypescriptClient: TypeScriptProject;
 
+  /**
+   * The directory in which the OpenAPI spec file(s) reside, relative to the project outdir
+   */
+  public readonly specDir: string;
+
+  /**
+   * The directory in which generated client code will be generated, relative to the outdir of this project
+   */
+  public readonly generatedCodeDir: string;
```

Comment:
This should be optional. Also please include`@default "generated"` to jsdocs.

Reply:
Ooh good catch!

Reply:
Ah hold on this is a property of the project and not an input, so is always available and readonly :)